packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -728,7 +728,11 @@ private void unlockAutoFocus() {
       // Cancel existing AF state.
       previewRequestBuilder.set(
           CaptureRequest.CONTROL_AF_TRIGGER, CameraMetadata.CONTROL_AF_TRIGGER_CANCEL);
-      captureSession.capture(previewRequestBuilder.build(), null, backgroundHandler);
```

Comment:
@letsar @gerken-tss Why delete this call? I'm not super clear why it was there initially, but I just want to make sure there's nothing I'm missing here.

Reply:
My guess is that it is not needed as we already have a call to `captureSession.capture(previewRequestBuilder.build(), null, backgroundHandler);` on line `741`. It makes sense to me to first configure the `previewRequestBuilder` with all the needed options before making the call. Can we please backport this fix to version `^0.10.0+3` and onward? this will allow flutter 2.10.x users to also get this fix.

Reply:
My guess is that this change is causing a breakage for [this test](https://github.com/flutter/plugins/blob/b89e4fc2dbd0e496de9ef0eeae92fdd1ca613d4c/packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/CameraTest.java#L665). Can you fix this to verify deleting this is correct?

In terms of back-porting the fix, I think we'd typically approach an issue like this with a forward fix like by landing this PR.